docs/JSON-RPC.md

```diff
@@ -29,6 +29,9 @@ The JSON-RPC server is based on the [JSON-RPC 2.0](https://www.jsonrpc.org/speci
 - A **response** from Jamulus to the consumer.
 - A **notification** from Jamulus to the consumer.
 
+**Tip:** On Linux, you can connect to a JSON-RPC server using the `nc` CLI tool. On Windows, you can download and use the `ncat` CLI tool.  This snippet uses [jayson](https://www.npmjs.com/package/jayson) to connect using Node.js:
+
+
```

Comment:
```suggestion
<pre>const jayson = require("jayson/promise");
const client = new jayson.client.tcp({ host: "127.0.0.1", port: 22100 });

client.request('jamulusserver/getServerInfo', {})
.then(console.log)
.catch(console.error)</pre>
```

Reply:
Perhaps I've reached a limit in the suggestion interface, by trying to show code there. 

In any case, this doesn't go here anyway. It gets dropped into the script source. But how do I designate the code? With **pre** or **code** tags?

Reply:
None of both. You'll need to use backticks: \`